cf/developer-access-group.yaml

```diff
@@ -169,8 +170,8 @@ Resources:
                   - "iam:ListMFADevices"
                   - "iam:ResyncMFADevice"
                 Resource:
-                  - "arn:aws:iam::*:user/${aws:username}"
-                  - "arn:aws:iam::*:group/*"
+                  - !Sub "arn:aws:iam::${AWS::AccountId}:user/${!aws:username}"
```

Comment:
what does this extra `!` in `${!aws:username}` do?

Reply:
To escape the `${aws:username}` which is a policy variable: 
https://stackoverflow.com/questions/44458304/how-to-escape-in-cloudformations-fnsub